tests/test_zip_endpoint.py

```diff
@@ -43,9 +43,14 @@ def setup_method(self):
         self.extended_scope_token = create_authz_token(
             [settings.BOUWDOSSIER_READ_SCOPE, settings.BOUWDOSSIER_EXTENDED_SCOPE]
         )
+        get_container_client(settings.STORAGE_ACCOUNT_CONTAINER_NAME)  # Call to create the container
+        self.queue_client = get_queue_client()
+        # Clear the queue to start with a clean slate
+        for message in self.queue_client.receive_messages(max_messages=100):
+            self.queue_client.delete_message(message)
```

Comment:
Do we need the visibility timeout in the tests?

Reply:
Nope. Because all messages are processed instantly.